yarn-project/aztec-rpc/src/aztec_rpc_server/aztec_rpc_server.ts

```diff
@@ -239,12 +247,13 @@ export class AztecRPCServer implements AztecRPCClient {
 
     const executionRequest = await this.getExecutionRequest(account, functionName, args, to);
 
-    // TODO: Can we remove tx context from this call?
-    const authedTxRequest = await entrypoint.createAuthenticatedTxRequest([executionRequest], TxContext.empty());
+    const txContext = TxContext.empty(await this.node.getChainId(), await this.node.getVersion());
+    const authedTxRequest = await entrypoint.createAuthenticatedTxRequest([executionRequest], txContext);
     if (!authedTxRequest.functionData.isPrivate) {
       throw new Error(`Public entrypoints are not allowed`);
     }
 
+    // Is simulating it messing it up?
```

Comment:
is this stale or?

Reply:
![Season 7 Oops GIF by Workaholics](https://media2.giphy.com/media/3ohzdYJK1wAdPWVk88/200.gif)